tests/plugins/a11yhelp/manual/commands.md

```diff
@@ -0,0 +1,52 @@
+@bender-tags: find, tc, 4.7.1, 16980
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea,sourcearea,htmlwriter,entities,toolbar,elementspath,undo,clipboard,format,basicstyles,autolink,autoembed,link,a11yhelp,pastetext
+
+### Scenario:
+
+1. Open Accessibility Help `ALT+0`.
+2. Scroll down to `Commands` section.
+
+### Expected:
+1. Mac users should see `Command` instead of `Ctrl` in instructions:
```

Comment:
You're already providing an information that there's be `command` key listed below. So this line should be simplified down to "Mac users should see:"